knowit/__init__.py

```diff
@@ -24,3 +24,8 @@
     from ordereddict import OrderedDict
 
 from .api import know
+
+
+_all = ['api', 'know', 'properties', 'provider', 'providers', 'utils', 'VIDEO_EXTENSIONS']
+
+__all__ = [a.encode('ascii') for a in _all]
```

Comment:
Not really, i added it we can use from knowit import * without pollution the namespace.

The encode ascii is required because of unicode_litterals, Unicode import fails